src/openforms/submissions/tests/factories.py

```diff
@@ -110,10 +117,40 @@ def from_components(
             name=f"definition-{key}", configuration=configuration
         )
         form_step = FormStepFactory.create(form=form, form_definition=form_definition)
+        form_variables = []
+        for component in iter_components(configuration=configuration, recursive=True):
+            if is_layout_component(component):
+                continue
+
+            form_variables.append(
+                FormVariableFactory.create(
+                    key=component["key"],
+                    is_sensitive_data=component.get("isSensitiveData", False),
+                    form=form,
+                    form_definition=form_definition,
+                    source=FormVariableSources.component,
+                )
+            )
+
```

Comment:
this pattern is repeated quite often in different places, can't we define a manager method? something like:

```py
FormVariable.objects.create_for(form)
```

That could be used in factories, tests and in actual application code if needed? Seems useful for the data-migration for existing forms, for example.